src/mixins/people-roles-mixin.ts

```diff
@@ -178,19 +178,15 @@ export default class PeopleRolesMixin extends Vue {
     return orgPerson?.roles.some(role => role.roleType === RoleTypes.PROPRIETOR)
   }
 
+  /** Called by ListPeopleAndRoles component event. */
   protected onEditPerson (index: number): void {
     this.currentOrgPerson = { ...this.orgPersonList[index] }
     this.activeIndex = index
     this.showOrgPersonForm = true
   }
 
+  /** Called by (Reg)AddEditOrgPerson component event. */
   protected onAddEditPerson (person: OrgPersonIF): void {
-    // if this is the completing party, assign email address from user profile
-    // NB: email cannot be null or empty
-    if (this.isCompletingParty(person) && !!this.getTombstone.userEmail) {
-      person.officer.email = this.getTombstone.userEmail
-    }
```

Comment:
This is where the email address was set AFTER the edit operation. I moved it because I needed the email adddress set before.